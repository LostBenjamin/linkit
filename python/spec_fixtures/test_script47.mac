

[[template A B C
  red
  A,B,C
]]

((template `1 + 1` `int(True and True)` ``))

