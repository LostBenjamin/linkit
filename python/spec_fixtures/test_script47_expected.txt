red:2,1

